offer-manager-domain/src/test/groovy/life/qbic/portal/portlet/offers/OfferSpec.groovy

```diff
@@ -200,7 +200,7 @@ class OfferSpec extends Specification {
         List<ProductItem> items = [
                 primaryAnalysisItem,
                 new ProductItem(1, new ProjectManagement("Basic Management",
-                        "Just an example", 10.0, 10.0, ProductUnit.PER_DATASET, 1, Facility.PCT))
+                        "Just an example", 10.0, 10.0, ProductUnit.PER_DATASET, 1, Facility.CFMB_PCT))
```

Comment:
Why did the naming change here?

Reply:
PCT enum was removed, since it is not required (Stakeholder Feedback)